targets/challenge/challenge-textures.qrc

```diff
@@ -13,6 +13,6 @@
 	<file>doc/challenge.fr.html</file>
 	<file>doc/challenge.es.html</file>
 	<file>doc/challenge.it.html</file>
-	<file>textures/asebachallenge.svgz</file
+	<file>textures/asebachallenge.svgz</file>
```

Comment:
I still don't understand why this line is here, maybe rebasing your branch on master remove this line from the patch?